### PR TITLE
Issue #13345: Enabled example tests for IllegalType

### PR DESCRIPTION
**Pull request by akankshat05**
For #13345

### CODE REVIEW

**Comment by romani** [comment]
GitHub, generate website

**Comment by romani** [comment]
please apply this patch https://dev.azure.com/romanivanovjr/romanivanovjr/_build/results?buildId=24539&view=logs&j=6a833393-c551-5000-0916-5000fd3b86e7&t=bdbbd129-77f0-5d73-ce7d-db58d34c8358&l=45
to let CI be green.

**Review by romani — CHANGES_REQUESTED** [review]
I confirm that examples are very big and complicated and I do not see easy way to make them simple.

the only items to improve:

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegaltype/Example1.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  abstract class A {
+
+    public abstract Set<Boolean> shortName(Set<? super Boolean> a);
+
+  }
+
+  class B extends Gitter {
+  }
+  class C extends Github {
+  }
```
[the remark this answers is not in the document]
Comment (akankshat05) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegaltype/Example1.java:63]: Okay, I will make the change

[the remark this answers is not in the document]
Comment (akankshat05) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegaltype/Example1.java]: Done

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegaltype/Example5.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="IllegalType">
+      <property name="memberModifiers" value="LITERAL_PUBLIC,
+        LITERAL_PROTECTED, LITERAL_STATIC"/>
```
[the remark this answers is not in the document]
Comment (akankshat05) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegaltype/Example5.java]: Done

**Review by romani — APPROVED** [review]
thanks a lot .

ok to merge.

**Pull request merged**